OpenSim/Tools/Test/vis_BuiltinGeometry.txt

```diff
@@ -118,9 +118,3 @@ DecorativeLine:~[0,0.05,0.35]~[0.19999,0.0480388,0.05] bodyId:0 color:~[0.05,0,0
 [0,0,1,0]
 [0,0,0,1]
 
-DecorativeMeshFile:blockMesh.obj  bodyId:1 color:~[0,1,1] indexOnBody:-1 Opacity:1 Rep:2 Scale:~[-1,-1,-1] Transform:
-[1,0,0,0]
-[0,1,0,0]
-[0,0,1,0]
-[0,0,0,1]
```

Comment:
As explained above the current Mesh display was misleading, I didn't want to make major change to testVisualization until my other PRs that modify it are merged in.  It maybe best to beef up the test when all ContactGeometry, and WrapGeometry are finalized instead of continuously changing/updating the tests but thanks for pointing out the potential hole in the current test.